src/manage-apps/fix-packages.md

```diff
@@ -0,0 +1,68 @@
+# Cleaning Up Packages with Terminal Commands
+
+If your system complains about a failed upgrade, package manager conflicts, broken upgrades, or other package-related issues, there are several common fixes to these problems. Some package manager issues can be resolved with the graphical update program, but many require the command line.
+
+## General Fixes for Apt Packages
+
+If a package was installed with `apt`, these commands perform general cleanup that can resolve many of `apt`'s errors. These commands should be run one at a time.
```

Comment:
I'm failing to see the connection between "If a package was installed with `apt`" and the rest of the sentence. Could that clause be omitted? It seems like what it really means is "If you see errors messages related to `apt` or `dpkg`."

Reply:
Removed clause